resources/css/style.css

```diff
@@ -85,3 +85,6 @@
   border: 0;
   z-index: 9;
 }
+.status {
+  height: fit-content;
```

Comment:
Support is not really good, only works in Chrome.
https://developer.mozilla.org/en-US/docs/Web/CSS/height